### ISSUE TITLE
Advanced rocketry Bug evasion through config changes (Not working)

### ISSUE THREAD

**Issue by amice101** [zmaster587/AdvancedRocketry#1376]
## Version of Advanced Rocketry
AdvancedRocketry-1.12.2-1.4.0-88-universal (Got from client)
## Have you verified this is an issue in the latest unstable build
?
## Version of LibVulpes
LibVulpes-1.12.2-0.2.8-32-universal (Got from client)
## Version of Minecraft
1.12.2
## Crash report or log (if applicable)
http://pastebin.com is a good place to put them
crash reports that are put in the issue itself are
hard to read
https://pastebin.com/4PR680Uu
## Current Modpack: 
Continuum (v1.4.1 MC Version 1.12.2)

## Description of the problem
[How can you reliably reproduce the problem]
I'm trying to disable discovery for planets on my server as there seems to be a chain of bugs associated with it on multiplayer, I've changed this {} in the config to turn off discovery but went back on the server only to find nothing has changed all the planets still require discovery/Not unlocked at all.

To sum it up/My aim
Due to the bugs I want to have the planets automatically discovered to cut out the bugs to enable progression through the mod

**Comment by amice101** [comment]
https://pastebin.com/7vK7xhJk

**Comment by zmaster587** [comment]
I don't know what you mean you changed {}, if you want to turn of discovery, you need to set planetsMustBeDiscovered to false in the config

**Comment by dmodoomsirius** [comment]
Is this still a issue? Please comment if it is and i will reopen if if you are unable? We are closing old issues to see what is still relevant and needs to be fixed.
